### PR TITLE
Feature/logos  and  img

### PR DESCRIPTION
**Pull request by Zara-Gma**
Completed feature: Logos and images.

### CODE REVIEW

**Comment by jordaniza** [comment]
@Zara-Gma - can you take a look at the merge conflicts?

**Comment by Zara-Gma** [comment]
I've updated the index file and Bankless.ts it's the same.

**Comment by Zara-Gma** [comment]
> @Zara-Gma - can you take a look at the merge conflicts?

@jordaniza I've updated the index file and Bankless.ts it's the same.

File: packages/react-app/src/constants/Bankless.ts [hunk 1]
```diff
@@ -1,10 +1,17 @@
-import { CustomerProps } from '../models/Customer';
+import { CustomerProps } from "../models/Customer";
 /**
```
Comment (jordaniza) [inline packages/react-app/src/constants/Bankless.ts:2]: Might want to be careful about linting changes like this - it introduces unecessary code changes

File: packages/react-app/src/components/global/Header/index.tsx [hunk 2]
```diff
@@ -1,40 +1,41 @@
...
+  const { customer } = useContext(CustomerContext);
+  return (
+    <NavBarContainer {...props}>
+      {!isOpen ? (
+        <Image
+          src={customer?.customization?.logo ?? '/logo.png'}
+          height={50}
+          width={50}
+          alt={`${customer?.customerName ?? 'DAO'} Logo`}
+        />
```
Comment (jordaniza) [inline packages/react-app/src/components/global/Header/index.tsx:33]: perfect

File: mongo/customers/customers_queries.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  [
+    {
+      $set: {
+        customization: {
+          logo: "/citydao.png",
+        },
+      },
+    },
+  ]
+);
```
Comment (jordaniza) [inline mongo/customers/customers_queries.js:16]: I'm assuming this only overrwrites a single field

**Comment by jordaniza** [comment]
Good stuff!

I'm going to merge this into develop AFTER the current release tomorrow. This means we can update logos next week. I'm happy to approve this work.

**Pull request closed without merging**